_includes/mobile/rewarded-server-side-configuration.md

```diff
@@ -0,0 +1,68 @@
+##### Server-side Rewarded Ad Unit Configuration
+
+The Rewarded Ad Unit assumes special behavior that should be configurable by the platform or publisher according to the application or ad experience guides.  
+
+Configuration of rewarded ad unit can be done using stored impression-level stored request and the [passthrough](https://docs.prebid.org/prebid-server/endpoints/openrtb2/pbs-endpoint-auction.html#request-passthrough) feature of Prebid Server. 
+
+Prebid SDK will search for a particular `rwdd` object in `ext.prebid.passthrough` of bid response to configure the behavior of the Rewarded Ad Unit. The following table describes the structure and usage purpose of `rwdd` configuration parameters.  
```

Comment:
what does the SDK actually do with these values? Is it for rendering?

Reply:
Yes, they are used by the rendering module to customize the appearance and behavior according to the parameters received in the response.